src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/auth/register/DetachedRegisterPage.java

```diff
@@ -0,0 +1,49 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.auth.register;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.signin.DetachedSignInPage;
+
+public class DetachedRegisterPage extends BasePageObject implements RegisterPage {
+
+  private RegisterPage registerPage;
+
+  public DetachedRegisterPage() {
+    this.registerPage = new AttachedRegisterPage();
+  }
+
+  public DetachedRegisterPage(RegisterPage registerPage) {
+    this.registerPage = registerPage;
+  }
+
+  @Override public AttachedRegisterPage open() {
+    return null;
+  }
+
+  @Override public AttachedRegisterPage typeEmailAddress(String email) {
+    return null;
+  }
+
+  @Override public AttachedRegisterPage typeUsername(String username) {
+    return null;
+  }
+
+  @Override public AttachedRegisterPage typePassword(String password) {
+    return null;
+  }
+
+  @Override public AttachedRegisterPage typeBirthdate(String month, String day, String year) {
+    return null;
+  }
+
+  @Override public void clickSignUpSubmitButton() {
+
+  }
+
+  @Override public boolean doesErrorMessageContainText() {
+    return false;
+  }
+
+  @Override public void verifyBirthdateError() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1186)